### ISSUE TITLE
Error in private variables strings with Angular 12

### ISSUE THREAD

**Issue by tomavic** [thevladeffect/ngx-typed-js#25]
Hello I am using Angular 12 with ngx-typed-js and I am getting 

```typescript
Error: src/app/app.component.html:34:28 - error TS2341: Property 'strings' is private and only accessible within class 'NgxTypedJsComponent'.

34             <ngx-typed-js [strings]="['asasas', 'asasdasdasfa']">
                              ~~~~~~~

  src/app/app.component.ts:6:16
    6   templateUrl: './app.component.html',
                     ~~~~~~~~~~~~~~~~~~~~~~
    Error occurs in the template of component AppComponent.

```

However your compiled component, says it's a private too!! 

![image](https://user-images.githubusercontent.com/17650005/125162365-b47f2c80-e187-11eb-875a-9202b70ba4b9.png)

Could you please fix this?

**Comment by mithunshanbhag** [comment]
Hitting this same issue with Angular 12.1.1.

This seems to be AOT/production related. There is a good explanation of the underlying error in [this stackoverflow thread](https://stackoverflow.com/questions/43141576/property-x-is-private-and-only-accessible-within-class-xyzcomponent).

**Comment by steven274** [comment]
remove 
```
    "strictInputAccessModifiers": true,
```
in tsconfig.json temporary fix this

**Comment by tomavic** [comment]
@steven274 yes, but this needs to be done from here too. Is this task hard to be done or will take too much time?

**Comment by riapacheco** [comment]
Also experiencing this

**Comment by thevladeffect** [comment]
This should be fixed in 2.1.1
